CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+
+# Contributing to As
+Thank you for considering contributing to As!
+
+## Reporting Bugs
+If you discover a bug, please create a [new issue](https://github.com/massalabs/as/issues/new?assignees=&labels=issue%3Abug&template=bug.md&title=) on our GitHub repository.
+In your issue, please include a clear and concise description of the bug, any relevant code snippets, error messages, and steps to reproduce the issue.
+
+## Installation
+To start developing with As, you must install all the necessary dev dependencies. You can do so by running the following command:
+
+```sh
+npm install
```

Comment:
They should also do a ```npm run build``` or the test won't pass

Reply:
I added npm run build in the tests section